packages/editor-ui/src/modules/credentials.ts

```diff
@@ -97,8 +97,21 @@ const module: Module<ICredentialsState, IRootState> = {
 			};
 		},
 		getCredentialsByType: (state: ICredentialsState, getters: any) => { // tslint:disable-line:no-any
+			let cache: {[type: string]: ICredentialsResponse[]} = {};
```

Comment:
a better approach would be to build a getter that returns this mapping.. it will automatically be updated when the original mapping is created.